forge-gui/res/cardsfolder/upcoming/imoen_trickster_friend.txt

```diff
@@ -0,0 +1,75 @@
+SVar:TrigTap:AB$ Tap | Cost$ ExileFromGrave<1/Instant;Sorcery> | ValidTgts$ Creature | SubAbility$ DBPump
+SVar:DBPump:DB$ Pump | Defined$ Targeted | Duration$ Permanent | KW$ HIDDEN This card doesn't untap during your next untap step.
+DeckHints:Ability$Graveyard & Type$Instant|Sorcery
+Oracle:Imoen, Wily Trickster can't be blocked.\nWhenever Imoen deals combat damage to a player, you may exile an instant or sorcery card from your graveyard. When you do, tap target creature an opponent controls. That creature doesn't untap during its controller's next untap step.
+
+SPECIALIZE:BLACK
+
+Name:Imoen, Occult Trickster
+ManaCost:1 W U
+Types:Legendary Creature Human Rogue Wizard
+PT:3/2
+K:Unblockable
+T:Mode$ DamageDone | ValidSource$ Card.Self | OptionalDecider$ You | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigToken | TriggerZones$ Battlefield | TriggerDescription$ Whenever NICKNAME deals combat damage to a player, you may exile an instant or sorcery card from your graveyard. If you do, create a 2/2 black Zombie creature token.
+SVar:TrigToken:AB$ Token | Cost$ ExileFromGrave<1/Instant;Sorcery> | TokenScript$ B_2_2_zombie
+DeckHas:Ability$Token & Type$Zombie
+DeckHints:Ability$Graveyard & Type$Instant|Sorcery
+Oracle:Imoen, Occult Trickster can't be blocked.\nWhenever Imoen deals combat damage to a player, you may exile an instant or sorcery card from your graveyard. If you do, create a 2/2 black Zombie creature token.
+
+SPECIALIZE:RED
+
+Name:Imoen, Chaotic Trickster
+ManaCost:1 U R
+Types:Legendary Creature Human Rogue Wizard
+PT:3/2
+K:Unblockable
+T:Mode$ DamageDone | ValidSource$ Card.Self |  ValidTarget$ Player | CombatDamage$ True | Execute$ TrigDamage | TriggerZones$ Battlefield | TriggerDescription$ Whenever NICKNAME deals combat damage to a player, you may exile an instant or sorcery card from your graveyard. If you do, NICKNAME deals 2 damage to each opponent.
+SVar:TrigDamage:AB$ DealDamage | Cost$ ExileFromGrave<1/Instant;Sorcery> | Defined$ Player.Opponent | NumDmg$ 2
+DeckHints:Ability$Graveyard & Type$Instant|Sorcery
+Oracle:Imoen, Chaotic Trickster can't be blocked.\nWhenever Imoen deals combat damage to a player, you may exile an instant or sorcery card from your graveyard. If you do, Imoen deals 2 damage to each opponent.
+
+SPECIALIZE:GREEN
+
+Name:Imoen, Wise Trickster
+ManaCost:1 G U
+Types:Legendary Creature Human Rogue Wizard
+PT:3/2
+K:Unblockable
+T:Mode$ DamageDone | ValidSource$ Card.Self | OptionalDecider$ You | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigDraw | TriggerZones$ Battlefield | TriggerDescription$ Whenever NICKNAME deals combat damage to a player, you may exile an instant or sorcery card from your graveyard. If you do, draw a card and you may play an additional land this turn.
+SVar:TrigDraw:AB$ Draw | Cost$ ExileFromGrave<1/Instant;Sorcery> | AddStaticAbility$ Exploration
```

Comment:
You can't just put AddStaticAbility$ onto a random effect and hope it will work :P